ci/jobs/scripts/integration_tests_runner.py

```diff
@@ -1215,7 +1216,15 @@ def run():
     if is_ci:
         # Dump dmesg (to capture possible OOMs)
         logging.info("Dumping dmesg")
-        subprocess.check_call("sudo -E dmesg -T", shell=True)
+        subprocess.check_call("sudo -E dmesg -T | tee dmesg.log", shell=True)
```

Comment:
Sorry for the late jump-in, but wouldn't it false-trigger on the previous OOM?

I think, we need to add here `--since $CHECK_START` to avoid historical OOMs

Reply:
We have

>INFO:root:Clearing dmesg before run

So this should not be an issue.